managed/services/inventory/grpc/services_server.go

```diff
@@ -268,3 +268,13 @@ func (s *servicesServer) RemoveService(ctx context.Context, req *inventorypb.Rem
 
 	return &inventorypb.RemoveServiceResponse{}, nil
 }
+
+// AddCustomLabels adds or replaces (if key exists) custom labels for a service.
+func (s *servicesServer) AddCustomLabels(ctx context.Context, req *inventorypb.AddCustomLabelsRequest) (*inventorypb.AddCustomLabelsResponse, error) {
```

Comment:
We has similar `grpc` package in managemet before, but we removed it. I guess we can do the same here if you have time for that.